src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import cl from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let currentId = activeTabId;
+
+  if (!tabs.find(tab => tab.id === activeTabId)) {
+    currentId = tabs[0].id;
+  }
+
+  function isSameId(id) {
+    if (activeTabId !== id) {
+      onTabSelected(id);
+    }
```

Comment:
The `isSameId` function should only call `onTabSelected` if the `id` is different from the `currentId`. Consider changing the condition to `if (currentId !== id)` to prevent unnecessary calls when the tab is already active.